protocol-designer/src/components/DeckSetup/LabwareOverlays/LabwareControls.tsx

```diff
@@ -1,28 +1,27 @@
-// @flow
 import * as React from 'react'
 import cx from 'classnames'
 import { RobotCoordsForeignDiv } from '@opentrons/components'
-import type { DeckSlot } from '@opentrons/shared-data'
+import { DeckSlot } from '@opentrons/shared-data'
 
-import { START_TERMINAL_ITEM_ID, type TerminalItemId } from '../../../steplist'
-import type { LabwareOnDeck } from '../../../step-forms'
+import { START_TERMINAL_ITEM_ID, TerminalItemId } from '../../../steplist'
+import { LabwareOnDeck } from '../../../step-forms'
 import { BlockedSlot } from './BlockedSlot'
 import { BrowseLabware } from './BrowseLabware'
 import { EditLabware } from './EditLabware'
 import { LabwareName } from './LabwareName'
 import { LabwareHighlight } from './LabwareHighlight'
 import styles from './LabwareOverlays.css'
 
-type LabwareControlsProps = {|
-  labwareOnDeck: LabwareOnDeck,
-  selectedTerminalItemId: ?TerminalItemId,
-  slot: DeckSlot,
-  setHoveredLabware: (?LabwareOnDeck) => mixed,
-  setDraggedLabware: (?LabwareOnDeck) => mixed,
-  swapBlocked: boolean,
-|}
+interface LabwareControlsProps {
+  labwareOnDeck: LabwareOnDeck
+  selectedTerminalItemId?: TerminalItemId | null
+  slot: DeckSlot
+  setHoveredLabware: (labware: LabwareOnDeck | null) => unknown
+  setDraggedLabware: (labware: LabwareOnDeck | null) => unknown
```

Comment:
```suggestion
  setDraggedLabware?: (labware: LabwareOnDeck | null) => unknown
```
These should remove the need for the `@ts-expect-errors` below